concepts/collateral-breakdown.md

```diff
@@ -0,0 +1,19 @@
+# Collateral Breakdown
+
+Collateral assets held by the Maker Protocol or its SubDAOs can be broken up into the following categories.
+
+{% hint style="warning" %} This documentation describes planned functionality and processes that MakerDAO has not yet implemented. Be aware that parts may be inaccurate or out of date. {% endhint %}
+
+## Decentralized collateral
```

Comment:
I'd probably just go into a bit more detail on these generally. I think Rune wrote at least a paragraph for each in the v3 forum post.